src/status_im/contexts/communities/actions/share_community/view.cljs

```diff
@@ -0,0 +1,89 @@
+(ns status-im.contexts.communities.actions.share-community.view
+  (:require
+    [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [react-native.fast-image :as fast-image]
+    [react-native.platform :as platform]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.qr-codes.view :as qr-codes]
+    [status-im.common.resources :as resources]
+    [status-im.contexts.communities.actions.share-community.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [{:keys [url community-id]}  (rf/sub [:get-screen-params])
+        window-width                (rf/sub [:dimensions/window-width])
+        {{:keys [thumbnail]} :images
+         color               :color
+         community-name      :name} (rf/sub [:communities/community community-id])
+        navigate-back               (rn/use-callback #(rf/dispatch [:navigate-back]))
+        on-press-share              (rn/use-callback
+                                     #(rf/dispatch
+                                       [:open-share
+                                        {:options (if platform/ios?
+                                                    {:activityItemSources
+                                                     [{:placeholderItem {:type    :text
+                                                                         :content url}
+                                                       :item            {:default {:type    :text
+                                                                                   :content url}}
+                                                       :linkMetadata    {:title (i18n/label
+                                                                                 :t/share-community)}}]}
+                                                    {:title     (i18n/label :t/share-community)
+                                                     :subject   (i18n/label :t/share-community)
+                                                     :message   url
```

Comment:
It's probably fine to not pass `url` as a dependency to `use-callback` here because probably `url` won't change while this view is mounted. But it's a bit like playing with fire and I prefer to always pass dependencies. One of the pain points of embracing these React hooks...

Reply:
Adding it to the dependency array.